src/components/FullHeader/FullHeader.js

```diff
@@ -54,6 +68,24 @@ const FullHeader = ({ user, cluster }) => (
 				) : null}
 			</Menu>
 		</div>
+		{isUsingTrial && (
+			<Button
+				css={trialBtn}
+				style={{
+					height: 'auto',
```

Comment:
We should move this to `css` prop. Also, it would be best if you use fixed height here - I'm not sure how adding `height: auto` is helping?